src/main/resources/static/js/reservation-with-member.js

```
@@ -0,0 +1,259 @@
+function createInput(type) {
+  const input = document.createElement('input');
+  input.type = type;
+  input.className = 'form-control';
+  return input;
+}
+
+function createActionButton(label, className, eventListener) {
+  const button = document.createElement('button');
+  button.textContent = label;
+  button.classList.add('btn', className, 'mr-2');
+  button.addEventListener('click', eventListener);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'createActionButton'. Consider to delete it or rename it.

'createActionButton' is defined here:


(lint/suspicious/noRedeclare)